.github/workflows/Updater-on-Demand.yml

```diff
@@ -58,7 +58,9 @@ jobs:
             git stash pop || true
             pkg=${i%.rb}
             pkg=${pkg#packages/}
-            pkg_version="$(LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH ruby tools/version.rb -j ${pkg} | jq -r '.[]|.version')"
+            LD_LIBRARY_PATH=/usr/local/lib ruby tools/version.rb -j ${pkg}
+            LD_LIBRARY_PATH=/usr/local/lib ruby tools/version.rb -j ${pkg} | jq -r '.[]|.version'
```

Comment:
Is line 61 necessary?

Reply:
Have you tried debugging locally with https://github.com/nektos/act? Might be simpler than pushing debugging code to master...

Reply:
@Zopolis4 I haven't tried that!